bom/pom.xml

```diff
@@ -25,7 +25,7 @@
 
   <groupId>io.netty</groupId>
   <artifactId>netty-bom</artifactId>
-  <version>4.1.82.Final</version>
+  <version>siren-4.1.82-1-SNAPSHOT</version>
```

Comment:
I forgot how things are done: you'll do a separate PR to release netty, once federate is shown to work with this ?

Reply:
Yes, I will do a separate pr for the release